src/classes/TestTrialDifficultyExtensionController.cls

```diff
@@ -0,0 +1,38 @@
+/**
+ * Created by hstoy on 10/16/2018.
+ */
+
+@isTest
```

Comment:
Tests must have assertions. Here you just execute the code, but not check if this code is doing what it supposed to.
U must check if output is according to the design of a method.